draft-ietf-quic-transport.md

```diff
@@ -2359,6 +2366,11 @@ genuine migrations.  Changing port number can cause a peer to reset its
 congestion state (see {{migration-cc}}), so the port SHOULD only be changed
 infrequently.
 
+Endpoints could also have their activity correlated if their peers keep using
+the same destination connection ID after migration. Nodes that initiate a
+migration need to provide their peers with new connection IDs. The goal is
```

Comment:
need is non-normative, which feels odd to me, and there's no mention of zero-length connection IDs.

Reply:
Yes. I rewrote that paragraph. Tell me what you think of the new version.